keps/sig-storage/20191125-bucket-provisioning.md

```diff
@@ -278,7 +279,8 @@ status:
     - `uuid`: the Bucket’s API server generated UUID
     - `resourceVersion`: the Bucket's resourceVersion
 1. `secretRef`: the name and namespace of the source secret.  This `secret` is either generated by the driver or created by an admin.
-1. `bucketIdentifier`: unique, identifiying information defined by the driver as an arbitrary string.  E.g. a bucket name, url, or serialized json object.  Base64 encoded to protect against syntax issues.
+1. `bucketNameActual`: the name of the bucket in the object store.
+1. `bucketIdentifier`: a unique value of set of values required by the driver to reach the bucket, optionally encoded by the driver into a serialized data format  Base64 encoded by sidecar.
```

Comment:
I assume we leave it up to drivers if they want to also include actual bucket name in this field.

Reply:
we can't force them not to, but we can make it a return value in the gRPC interface to at least encourage its use.  Depending on how we implement the system, apps would expected it be set (if we use it to name a file or env var for instance), further encouraging drivers authors to set it.